torch/linalg/__init__.py

```diff
@@ -140,6 +140,110 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+eigh = _add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
+
+This function returns eigenvalues and eigenvectors
```

Comment:
Does this function work on batches of these tensors, too?

Reply:
"This function returns" -> "Computes the"

Reply:
It works for batched input as well, that's tested and I updated the docs now.